client/layout/index.jsx

```diff
@@ -200,6 +201,10 @@ class Layout extends Component {
 		);
 	}
 
+	shouldShowHelpCenter() {
+		return !! getQueryArg( window.location.href, 'showHelpCenter' );
+	}
```

Comment:
This feels more like a utility function to me. WDYT about moving it to `utils.ts` and add a unit test?